thriftpy/protocol/compact.py

```diff
@@ -58,7 +58,10 @@ def write_varint(trans, n):
         else:
             out.append((n & 0xff) | 0x80)
             n = n >> 7
-    data = array.array('B', out).tostring()
+
+    # 3.2+ uses tobytes(). Thriftpy doesn't support Python 3 pre-3.3
+    a = array.array('B', out)
+    data = a.tobytes() if PY3 else a.tostring()
```

Comment:
That's a good idea, thanks @wvonnegut.  If one of the maintainers would prefer it that way, I'd be happy to change it, just let me know.